ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/websocket/RawWebSocket.kt

```diff
@@ -33,6 +33,10 @@ public class RawWebSocket(
     override val incoming: ReceiveChannel<Frame> get() = filtered
     override val outgoing: SendChannel<Frame> get() = writer.outgoing
 
+    @OptIn(ExperimentalWebSocketExtensionApi::class)
```

Comment:
This is declared as experimental but here we opt-in instead

Reply:
Sure, will fix